modules/bastion/variables.tf

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2017, 2023 Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+# Common
+variable "compartment_id" { type = string }
+variable "state_id" { type = string }
+
+# Bastion
+variable "assign_dns" { type = bool }
+variable "availability_domain" { type = string }
+variable "image_id" { type = string }
+variable "nsg_ids" { type = list(string) }
+variable "public" { type = bool }

```

Comment:
please keep the same variable names as in bastion's variables. This way it'll be easier to update

Reply:
Renamed to `is_public` for consistency.